scss/dependencies/_mixins.scss

```diff
@@ -33,7 +33,7 @@
 }
 
 @function pt-to-rem($pt) {
-  @return ($pt * 0.133) * 1rem;
+  @return ($pt * 0.133 * 0.625) * 1rem;
```

Comment:
`1px == 0.625rem` and `1pt == 1.33px`. We need to transform `pt` to `rem`. (Authored by @emersonfarias) :P